README.md

```diff
@@ -1,20 +1,71 @@
-# Noto | Minimal Note-Taking App
-
-<img src="images/Icon.svg" alt="icon"/>
+<h1 align="center">
+  <img src="images/Icon.svg" alt="icon"/>
+  <br>
+  <b>Noto | Minimal Note-Taking App</b>
+</h1>
+
+<div align="center">
+  <a>
+    <img src="https://img.shields.io/badge/Android-3DDC84?style=for-the-badge&logo=android&logoColor=white" alt="Android">
+  </a>
+  <a>
+    <img src="https://img.shields.io/badge/kotlin-%237F52FF.svg?style=for-the-badge&logo=kotlin&logoColor=white" alt="Kotlin">
+  </a>
+  <a href="https://www.patreon.com/alialbaali">
+    <img src="https://img.shields.io/badge/Patreon-F96854?style=for-the-badge&logo=patreon&logoColor=white">
+  </a>
+  <a href="https://www.buymeacoffee.com/alialbaali">
+    <img src="https://img.shields.io/badge/Buy%20Me%20a%20Coffee-ffdd00?style=for-the-badge&logo=buy-me-a-coffee&logoColor=black" alt="BuyMeACoffee">
+  </a>
+  <a href="/LICENSE.md">
+    <img src="https://img.shields.io/badge/Apache%202.0-green?style=for-the-badge&logo=apache" alt="License">
+  </a>
+</div>
+
+<p align="center">
+  <a href="#download">Download</a>
+  •
+  <a href="#motivation">Motivation</a>
+  •
+  <a href="#features">Features</a>
+  •
+  <a href="#contributions">Contributions</a>
+  •
+  <a href="#translation-and-localization">Translation & Localization</a>
+  •
+  <a href="#architecture">Architecture</a>
+  •
+  <a href="#screenshots">Screenshots</a>
+  •
+  <a href="#support">Support</a>
+  •
+  <a href="#license">License</a>
+</p>
 
 ## Download
 
-You can download Noto from [Github](https://github.com/alialbaali/Noto/releases/latest) or Google Play using the following link:
-
-[<img src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png" alt="Get it on Google Play" height="80">](https://play.google.com/store/apps/details?id=com.noto)
+<div align="center">
+  <h2><b>You can get Noto from one of the links below!</b></h2>
```

Comment:
Do you think this line necessary?

Reply:
Not necessarily, however it does look slightly better on the readme.

Reply:
Alright, but maybe smaller heading would be better, don't you think? It looks like two sections and the first section doesn't have anything in it.